tests/product-cart.spec.ts

```diff
@@ -0,0 +1,34 @@
+import { test, expect } from '@playwright/test';
+
+test.beforeEach(async ({ page }) => {
+    await page.goto(process.env.WEB_URL as string);
```

Comment:
```suggestion
    await page.goto('/');
```
because you added baseURL to the PW config file